lib/puppet/http/client.rb

```diff
@@ -129,8 +129,6 @@ def get(url, headers: {}, params: {}, options: {}, &block)
     execute_streaming(request, options: options) do |response|
       if block_given?
         yield response
```

Comment:
It'd probably be better to just do `execute_streaming(request, options: options, &block)`